src/devices/Board/AlternatePinMode.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Board
+{
+    /// <summary>
+    /// Alternate pin modes, for pins that support them
+    /// </summary>
+    public enum AlternatePinMode
+    {
+        // Note: Do not change the enum values without reviewing the impact
+        Unknown = -2,
```

Comment:
I saw you were using int directly instead of this enum, now the negative values are clearer, you should use this enum directly

Reply:
Oh, apparently I got confused myself. 

I think I'll try a class instead.